skopt/gp_opt.py

```diff
@@ -171,60 +182,60 @@ def gp_minimize(func, bounds, base_estimator=None, acq="UCB", xi=0.01,
     rng = check_random_state(random_state)
 
     # Bounds
-    num_params = len(bounds)
-    lower_bounds, upper_bounds = zip(*bounds)
-    upper_bounds = np.asarray(upper_bounds)
-    lower_bounds = np.asarray(lower_bounds)
-    lbfgs_bounds = np.tile((0, 1), (num_params, 1))
+    n_params = len(bounds)
+    lb, ub = extract_bounds(bounds)
 
     # Default GP
     if base_estimator is None:
         base_estimator = GaussianProcessRegressor(
-            kernel=Matern(length_scale=np.ones(num_params), nu=2.5),
-            normalize_y=True, random_state=random_state)
+            kernel=(ConstantKernel(constant_value_bounds="fixed") *
+                    Matern(length_scale=np.ones(n_params),
+                           length_scale_bounds="fixed", nu=2.5) +
+                    ConstantKernel(constant_value_bounds="fixed")),
```

Comment:
But for sure, it makes no sense now if it is fixed... We should either remove it or set tighther bounds as explained in #45